jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -154,28 +145,7 @@ bullet_pkg="libbullet-dev"
 
 # choose dart version
 if $DART_FROM_PKGS; then
-    if [[ ${GAZEBO_MAJOR_VERSION} -ge 9 ]]; then
-       dart_pkg="libdart6-utils-urdf-dev"
-    elif [[ ${GAZEBO_MAJOR_VERSION} -ge 8 ]]; then
-       dart_pkg="libdart-core5-dev"
-    fi
-fi
-
-# gazebo8 and above use qt5
-if [[ ${GAZEBO_MAJOR_VERSION} -le 7 ]]; then
-  gazebo_qt_dependencies="libqt4-dev \\
-                          libqtwebkit-dev"
-else
-  # After gazebo8 is released, these two lines should be all that remain
-  gazebo_qt_dependencies="qtbase5-dev \\
-                          libqwt-qt5-dev"
-  # Install qt4 as well for gazebo8 until its release
-  # 20170125 release date of gazebo8
-  if [[ $(date +%Y%m%d) -le 20170125 ]]; then
-    gazebo_qt_dependencies="${gazebo_qt_dependencies} \\
-                            libqt4-dev \\
-                            libqwt-dev"
-  fi
+  dart_pkg="libdart6-utils-urdf-dev"
```

Comment:
Does this work on Focal?

Reply:
Good point, I also thought on it while modifying that line.  The whole dart support needs fixing to reflect all the packages that needs Gazebo (11 needs at least collision-bullet collision-ode utils-urdf). I'll send a new PR to fix it. In this PR I wanted to keep same functionality but just remove old code.